src/core/server/server.cc

```diff
@@ -1809,12 +1813,18 @@ Server::CallData::CallData(grpc_call_element* elem,
                     elem, grpc_schedule_on_exec_ctx);
   GRPC_CLOSURE_INIT(&recv_trailing_metadata_ready_, RecvTrailingMetadataReady,
                     elem, grpc_schedule_on_exec_ctx);
+
+  server_->stream_quota_->IncrementOutstandingRequests();
```

Comment:
This is the callv1 request path. We'll need similar logic on the callv3 path (see MakeCallDestination)